### ISSUE TITLE
LIQUIDITY flowchart/roadmap

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#24]
fkinglag wants to do this until the next hangout (session 23)

mentioned in [hangout 22](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2017_01_21.MD):

**Comment by fkinglag** [comment]
Yes I plan on doing some work related on creating a flowchart along the lines of looking like an infographic. The current state of gridcoin with where we are connected on exchanges among other means in the growing economy will be listed. I am looking for any sort of input or feedback from anybody about this. More details and such will be posted soon as I put it together. Thanks!

**Comment by grctest** [comment]
@fkinglag Did you produce an updated infographic? I remember seeing a draft version on IRC a couple months ago..
